.github/workflows/maintainer_merge.yml

```diff
@@ -0,0 +1,54 @@
+name: Maintainer merge
+
+on:
+  issue_comment:
+    types: [created, edited]
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  ping_zulip:
+    name: Ping maintainers on Zulip
+    if: (toJSON(github.event.issue.pull_request) != 'null') && (startsWith(github.event.comment.body, 'maintainer merge') || contains(toJSON(github.event.comment.body), '\r\nmaintainer merge'))
```

Comment:
This second `toJSON` looks fishy to me; what's the reason for it?

Reply:
This was just copy pasted from https://github.com/leanprover-community/mathlib/blob/9f28b5d4deaba1f87b1b8f50a380570d5578c6fb/.github/workflows/add_label_from_comment.yml, I think.

Reply:
Hmm, I don't remember specifically having to put it there, so it might just have been a copy-paste error on my part.

Reply:
Update: after Johan's testing it turns out this toJSON is necessary, though the specific reason is still mysterious to me.